server/config/config.json

```diff
@@ -1,20 +1,20 @@
 {
   "development": {
-    "username": "root",
+    "username": "main",
     "password": null,
     "database": "jkitchen",
     "host": "127.0.0.1",
     "dialect": "postgres"
   },
   "test": {
-    "username": "root",
+    "username": "main",
     "password": null,
     "database": "jkitchentest",
     "host": "127.0.0.1",
     "dialect": "postgres"
   },
   "production": {
-    "username": "root",
+    "username": "main",
     "password": null,
     "database": "database_production",
     "host": "127.0.0.1",
```

Comment:
Please consider changing the file extension to .js, also your environment should not be multiple, you defined test, dev and production environment, this values should also be gotten from your  environment config. Your app should be able to adapt to its environment via the config file.